Makefile

```diff
@@ -49,7 +44,11 @@ license-check:
 	$(GOBIN)/go-licenses check $(PKG)
 
 test:
-	go test -cover ./...
+	bazel test //...
+
+update-bazel:
```

Comment:
`update-bazel` is misleading: we're not updating `bazel` but rather regenerating the `BUILD` files. Perhaps `build-gen` is a more descriptive name?

Reply:
Actually stole the name from kubernetes, but agree name is misleading

https://github.com/kubernetes/repo-infra/blob/master/hack/update-bazel.sh